Tests/SnapshotTests/Components/GiftCardComponentUITests.swift

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) @testable import Adyen
+@_spi(AdyenInternal) @testable import AdyenCard
+@testable import AdyenComponents
+import AdyenDropIn
+import XCTest
+
+class GiftCardUITests: XCTestCase {
```

Comment:
I'm a bit confused as I didn't expect to see Gift card tests in the UPI feature

Reply:
We didn't have UI Tests for the GiftCardComponent but we changed the look of a GiftCardComponent related item. Might have been it's own PR but also wanted to make sure this reflected as soon as possible

Reply:
See: https://github.com/Adyen/adyen-ios/pull/1554#discussion_r1668442518